libs/components/forms/src/lib/modules/radio/radio-group.component.scss

```diff
@@ -1,19 +1,86 @@
+@use 'libs/components/theme/src/lib/styles/variables' as *;
+@use 'libs/components/theme/src/lib/styles/compat-tokens-mixins' as compatMixins;
+
+@include compatMixins.sky-default-host-overrides() {
+  --sky-override-radio-group-hint-text-space: #{$sky-space-xs};
+  --sky-override-radio-group-content-space: #{$sky-space-sm};
+  --sky-override-radio-group-content-with-hint-text-space: #{$sky-space-md};
+
+  .sky-radio-group-legend {
+    .sky-radio-group-heading-text {
+      line-height: 1.1;
+    }
+  }
+}
+
+@include compatMixins.sky-default-host-overrides() {
```

Comment:
any reason for including this mixin twice?

Reply:
That was a mistake. The first one shouldn't have been a host override and should have been scoped.